commons/src/test/java/io/hyscale/commons/config/SetupConfigTests.java

```diff
@@ -0,0 +1,100 @@
+/**
+ * Copyright 2019 Pramati Prism, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.hyscale.commons.config;
+
+import io.hyscale.commons.constants.ToolConstants;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class SetupConfigTests {
+    private static final String APP_NAME = "myApp";
+    private static final String SVC_NAME = "mySvc";
+    private static final String HYSCALE = "hyscale";
+    private static final String GENERATED_FILES = "generated-files";
+    private static final String LOGS_DIRECTORY = "logs";
+    private static final String APPS_DIRECTORY = "apps";
+    private static StringBuilder appsDir;
+    private static StringBuilder serviceDir;
+    private static SetupConfig setupConfig;
+
```

Comment:
You can define appsDir and serviceDir as String and have them set in init() method using stringbuilder